### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ranguli**
## Summary

Added more commonly known Issues to the README.md, referencing Issue #12 and #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +127,18 @@ swap filename collision.
...
 
+### E492: Not an editor command: LLPStartPreview
+Check Issue [#12](https://github.com/xuhdev/vim-latex-live-preview/issues/12), provided the plugin is correctly installed, this is likely a **Python** issue.
+
+### Python-related Issues 
+
+* 2.7 vs 3.5 
+This is an ongoing issue: [#12](https://github.com/xuhdev/vim-latex-live-preview)
+
+* dyn/python
```
Comment (badouralix) [inline README.md]: It should be `python/dyn` instead of `dyn/python`

File: README.md [hunk 2]
```diff
@@ -127,6 +127,18 @@ swap filename collision.
 Currently, root file must be in the same directory or upper in the project tree
 (otherwise, one has to save file to update the preview).
 
+### E492: Not an editor command: LLPStartPreview
+Check Issue [#12](https://github.com/xuhdev/vim-latex-live-preview/issues/12), provided the plugin is correctly installed, this is likely a **Python** issue.
+
+### Python-related Issues 
+
+* 2.7 vs 3.5 
+This is an ongoing issue: [#12](https://github.com/xuhdev/vim-latex-live-preview)
```
Comment (badouralix) [inline README.md]: Unsound link

File: README.md (hunk 1 above)
Comment (ranguli) [inline README.md]: Thank you, fixed in the latest commit.

File: README.md (hunk 2 above)
Comment (ranguli) [inline README.md]: Fixed in latest commit :+1:

**Comment by badouralix** [comment]
I'm going to try the [`Squash and merge` option](https://help.github.com/articles/about-pull-request-merges/#squash-and-merge-your-pull-request-commits)

**Comment by ranguli** [comment]
Nice, that's a handy feature I didn't know about. :+1:

**Pull request merged**